bindata/oauth-openshift/deployment.yaml

```diff
@@ -0,0 +1,139 @@
+          effect: NoExecute
+          tolerationSeconds: 120
+      containers:
+        - name: oauth-openshift
+          image: ${IMAGE}
+          command:
+            - /bin/bash
+            - '-ec'
+          args:
+            - |
+              if [ -s /var/config/system/configmaps/v4-0-config-system-trusted-ca-bundle/ca-bundle.crt]; then
+                  echo "Copying system trust bundle"
+                  cp -f /var/config/system/configmaps/v4-0-config-system-trusted-ca-bundle/ca-bundle.crt `/etc/pki/ca-trust/extracted/pem`/tls-ca-bundle.pem
+              fi
+              exec oauth-server osinserver \
+              --config=/var/config/system/configmaps/v4-0-config-system-cliconfig/v4-0-config-system-cliconfig \
+              --v=${LOG_LEVEL}
+          ports:
+            - name: https
+              containerPort: 6443
+              protocol: TCP
+          volumeMounts:
+            - name: v4-0-config-system-session
+              readOnly: true
+              mountPath: /var/config/system/secrets/v4-0-config-system-session
+            - name: v4-0-config-system-cliconfig
+              readOnly: true
+              mountPath: /var/config/system/configmaps/v4-0-config-system-cliconfig
+            - name: v4-0-config-system-serving-cert
+              readOnly: true
+              mountPath: /var/config/system/secrets/v4-0-config-system-serving-cert
+            - name: v4-0-config-system-service-ca
+              readOnly: true
+              mountPath: /var/config/system/configmaps/v4-0-config-system-service-ca
+            - name: v4-0-config-system-router-certs
+              readOnly: true
+              mountPath: /var/config/system/secrets/v4-0-config-system-router-certs
+            - name: v4-0-config-system-ocp-branding-template
+              readOnly: true
+              mountPath: /var/config/system/secrets/v4-0-config-system-ocp-branding-template
+            - name: v4-0-config-system-trusted-ca-bundle
+              readOnly: true
+              mountPath: /var/config/system/configmaps/v4-0-config-system-trusted-ca-bundle
+          readinessProbe:
+            httpGet:
+              path: /healthz
+              port: 6443
+              scheme: HTTPS
+            timeoutSeconds: 1
+            periodSeconds: 10
+            successThreshold: 1
+            failureThreshold: 3
+          livenessProbe:
+            httpGet:
+              path: /healthz
+              port: 6443
+              scheme: HTTPS
+            initialDelaySeconds: 30
+            timeoutSeconds: 1
+            periodSeconds: 10
+            successThreshold: 1
+            failureThreshold: 3
+          terminationMessagePolicy: FallbackToLogsOnError
+          resources:
+            requests:
+              cpu: 10m
+              memory: 50Mi
+      volumes:
+        - name: v4-0-config-system-session
+          secret:
+            secretName: v4-0-config-system-session
+            defaultMode: 420
```

Comment:
this looks suspicious/wrong.  Aren't these supposed to be octal?

Reply:
420 decimal == 0644 octal, which is the default anyway, so I have removed these lines.